msrest/http_logger.py

```diff
@@ -43,6 +43,8 @@ def log_request(adapter, request, *args, **kwargs):
         _LOGGER.debug("Request method: %r", request.method)
         _LOGGER.debug("Request headers:")
         for header, value in request.headers.items():
+            if header == 'Authorization':
```

Comment:
This is an excellent idea! Actually, we do the same thing in the test framework to avoid saving on records this header... Should have been done from the start.
However, I would have written:
```python
if header.lower() == 'authorization':
    continue
```
Could you update your PR? At worst I will try to do a new one.
Thanks!!!!